themes/root/templates/RecordDriver/Primo/export-endnote.phtml

```diff
@@ -61,6 +61,6 @@ case 'Website':
     break;
 }
 
-$this->overrideFormats = [$endnoteFormat];
+$this->slot('endnote-formats')->set([$endnoteFormat]);
```

Comment:
I wonder if it's an oversight that we don't have endnoteweb versions of these templates for Primo and Summon....